wordcloud/wordcloud.py

```diff
@@ -233,8 +233,8 @@ def fit_words(self, words):
         self.layout_ = list(zip(words, font_sizes, positions, orientations, colors))
         return self.layout_
 
-    def process_text(self, text):
-        """Splits a long text into words, eliminates the stopwords.
+    def _get_words(self, text):
```

Comment:
Why did you remove this public function? Backward incompatible changes make it hard for others to use the code.